pkg/lightning/config/config.go

```diff
@@ -302,15 +302,16 @@ type FileRouteRule struct {
 }
 
 type TikvImporter struct {
-	Addr             string   `toml:"addr" json:"addr"`
-	Backend          string   `toml:"backend" json:"backend"`
-	OnDuplicate      string   `toml:"on-duplicate" json:"on-duplicate"`
-	MaxKVPairs       int      `toml:"max-kv-pairs" json:"max-kv-pairs"`
-	SendKVPairs      int      `toml:"send-kv-pairs" json:"send-kv-pairs"`
-	RegionSplitSize  ByteSize `toml:"region-split-size" json:"region-split-size"`
-	SortedKVDir      string   `toml:"sorted-kv-dir" json:"sorted-kv-dir"`
-	DiskQuota        ByteSize `toml:"disk-quota" json:"disk-quota"`
-	RangeConcurrency int      `toml:"range-concurrency" json:"range-concurrency"`
+	Addr               string   `toml:"addr" json:"addr"`
+	Backend            string   `toml:"backend" json:"backend"`
+	OnDuplicate        string   `toml:"on-duplicate" json:"on-duplicate"`
+	MaxKVPairs         int      `toml:"max-kv-pairs" json:"max-kv-pairs"`
+	SendKVPairs        int      `toml:"send-kv-pairs" json:"send-kv-pairs"`
+	RegionSplitSize    ByteSize `toml:"region-split-size" json:"region-split-size"`
+	SortedKVDir        string   `toml:"sorted-kv-dir" json:"sorted-kv-dir"`
+	DiskQuota          ByteSize `toml:"disk-quota" json:"disk-quota"`
+	RangeConcurrency   int      `toml:"range-concurrency" json:"range-concurrency"`
+	DuplicateDetection bool     `toml:"duplicate-detection" json:"duplicate-detection"`
```

Comment:
can we reuse the "on-duplicate" field?

Reply:
I had this idea at first, but considering that it would be a little strange. 😂
From my understanding, `on-duplicate` is used to tell lgihtning what should to do when encountered duplicates.
But for local backend duplicate detection, what we need is a switch which controls whether we should detect
duplicates and report errors.
I haven no idea yet on how to keep `on-duplicate` consistent with both tidb backend and local backend.

Reply:
okay